ts/Core/Utilities.ts

```diff
@@ -890,6 +893,10 @@ function wrap<T, K extends FunctionNamesOf<T>>(
 ): void {
     const proceed = obj[method] as T[K]&ArrowFunction;
 
+    if (typeof proceed !== 'function') {
+        throw new Error(`WrapError: ${method} is not a function`);
+    }
```

Comment:
Is this necessary? What changed? And if necessary, should we use an error code?

Reply:
PS: In v10, and previous, this silently fails. As can be seen in the wrap-drawgraph function. I prefer that, or a warning, over throwing an error.

Reply:
It helped to identify the errors with ES6 prototype. I can remove it.